src/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+
+function Greeting() {
+  const [randomGreeting, setRandomGreeting] = useState({ text: '' });
+
+  useEffect(() => {
+    fetch('http://127.0.0.1:3000/api/random_greeting')
+      .then((response) => response.json())
+      .then((data) => setRandomGreeting(data));
+  }, []);
+
+  return (
+    <div>
+      <h1>Random Greeting:</h1>
+      <p>{randomGreeting.text}</p>
+    </div>
+  );
+}
```

Comment:
- [ ] @AlejandraKeber Great work on your app however the app doesn't display the greeting message as shown below, I also noted you haven't used the data fetched from the redux store in your greeting component as per the project requirement. Kindly consider using the data from the redux store and ensure the random message is getting displayed
![image](https://github.com/AlejandraKeber/hello-react-front-end/assets/61408860/a643855e-dcfd-4703-821e-ebe2d4025b47)